docs/source/howto/codes.rst

```diff
@@ -45,8 +45,90 @@ Design guidelines
 Running external codes
 ======================
 
-`#3987`_
+To run an external code with AiiDA, you will need to use an appropriate :ref:`calculation plugin <topics:plugins>` that knows how to transform the input nodes into the input files that the code expects, copy everything in the code's machine, run the calculation and retrieve the results.
+This plugin would have already been created, either by you (see section on :ref:`interfacing external codes <how-to:codes:plugin>`) or by other AiiDA plugin-developers (available packages are listed in the `plugin registry <https://aiidateam.github.io/aiida-registry/>`_), and has been properly installed in your environment.
```

Comment:
```suggestion
You can check the `plugin registry <https://aiidateam.github.io/aiida-registry/>`_ to see if a plugin already exists for the code that you would like to run.
If that is not the case, you can :ref:`develop your own <how-to:codes:plugin>`.
After you have installed the plugin, you can start running the code through AiiDA.
```
Just wondering if we need a quick link for "installed the plugin" to a section on how to install plugins. Not sure if we already have it.

Reply:
Yeah, I thougth the same, but there is currently no such section. Should it be part of "How to manage your installation" maybe?

Reply:
Yes make a section stub there called "Installing plugins" and create the corresponding issue, if you will please. Put it before the "Tuning performance" section, for now